### ISSUE TITLE
Whether it depends on laser ranging

### ISSUE THREAD

**Issue by kubbo** [Theta-Limited/OpenAthenaAndroid#101]
Does this project necessarily rely on laser ranging

**Comment by mkrupczak3** [comment]
No, this project uses terrain-raycasting, which precludes the need for laser ranging. Think of it as simulating a laser ranging using a digital twin of the Earth as represented by a digital elevation model. This allows even common drones with inexpensive sensors to determine precise ground locations
